### ISSUE TITLE
Project Status

### ISSUE THREAD

**Issue by newpIDC** [mame82/P4wnP1#254]
Great project keep it up. However, Its been about 4 months since active development on this. Is it dead? is someone taking over? Please keep it alive.

**Comment by mame82** [comment]
Thanks for your question.
No worries, I'm working on P4wnP1... there have been 60 commits in the last 4 month.
The reason nobody could see it: I started rewroting the project from scratch in go.
There're a bunch of features planned (and already working), which force me to redesign many things, this takes time.
Among others, these are:

- a new, more powerful HID scripting language
- runtime re-configuration of USB gadgets
- additional gadget support (CD-Rom, serial ...)
- multiplatform remote CLI configuration tool
- web frontend with reactive settings control

So everything is kept in a private repo, till I reached the point all basic features are working without major changes.

To keep everybody up to date on what's going on, I use twitter (@P4wnP1 @Mame82).

... and no, there's no planned release date ... sorry
